docs/metadata.rst

```diff
@@ -0,0 +1,98 @@
+--------------
+
+-  Documentation:
+
+   -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html
+
+   -  `https://docs.openmicroscopy.org/omero/latest/users/cli/index.html <https://docs.openmicroscopy.org/omero/latest/users/cli/index.html>`__
+
+-  Data: example images from
+
+   -  IDR data http://idr.openmicroscopy.org/webclient/?show=project-51
+
+-  Metadata plugin for OMERO
+
+   - https://pypi.org/project/omero-metadata/
+
+-  Bulkmap config yml files defining the various Key-Value pairs parameters, such as the groups and other parameters.
+
+   - https://github.com/IDR/idr0021-lawo-pericentriolarmaterial/blob/9479af85f19487f215e3dfdd31a1b587370ed3cf/experimentA/idr0021-experimentA-bulkmap-config.yml
+
+-  Annotation csv file defines the content of OMERO.tables for each image.
+
+   - https://github.com/IDR/idr0021-lawo-pericentriolarmaterial/blob/9479af85f19487f215e3dfdd31a1b587370ed3cf/experimentA/idr0021-experimentA-annotation.csv
+
+Setup:
+------
+
+**Metadata plugin installation**
+
+- Go to the environment where you installed your OMERO.cli as specified under -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html.
+
+- Activate the virtual environment.
+
+- Run::
+    
+    $ pip install omero-metadata
+
+**Step-by-step:**
+-----------------
+
+#.  On your local machine, open a terminal
+
+#.  Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.::
+
+    $ export PATH=/opt/omero/server/venv3/bin:$PATH
+
+#.  Download the csv from https://github.com/IDR/idr0021-lawo-pericentriolarmaterial/blob/9479af85f19487f215e3dfdd31a1b587370ed3cf/experimentA/idr0021-experimentA-annotation.csv 
+
+#.  The variable ``$ID​`` below is the ID of the ​Project, in this example case it is the Project containing the idr0021 study. To add annotations from a local csv file to the images in the said Project in the form of OMERO.tables, run::
+    
+    $ omero metadata populate --report --batch 1000 --file local/path/to/idr0021-experimentA-annotation.csv Project:$ID
+
+#.  Open your browser and login to the OMERO.web. Navigate to the Project you just worked with, expand the "Attachments" harmonica in the right-hand pane and verify that a new attachment is on that Project named ``bulk_annotations``.
+
+#.  You can inspect its content by clicking on the "eye" icon inside the annotation.
+
+#.  Select an image inside the Project and expand the "Tables" harmonica in the right-hand pane. These tables contain the appropriate line from the ``bulk_annotations`` attachment you just created for that particular image.
+
+#.  Go back to your terminal. Download the idr0021-experimentA-bulkmap-config.yml file https://github.com/IDR/idr0021-lawo-pericentriolarmaterial/blob/9479af85f19487f215e3dfdd31a1b587370ed3cf/experimentA/idr0021-experimentA-bulkmap-config.yml.
+
+#.  Open the downloaded file in a text editor and delete the ``Advanced options...`` section. Save the file and run::
```

Comment:
I think it would be clearer if you bundled a modified copy of that file in this repository instead of telling people to download an external file and modify it.

Reply:
It's an interesting question. Benefits of using the original would include:
 - no need to keep changes synchronized
 - teaching users that there's a wealth of examples under GitHub.com/idr
Benefits of a copy would include:
 - clarity as you say
 - Ability to perhaps simplify the file

cc: @sbesson

Reply:
Do you anticipate needing to modify the released idr0021 study?

Reply:
@joshmoore @manics Please note https://github.com/ome/omero-metadata/issues/44 - this is the main reason why not to use the idr studies workflows as they are on outreach.